gensrc/proto/lake_types.proto

```diff
@@ -116,11 +120,17 @@ message TxnLogPB {
         optional DelvecMetadataPB delvec_meta = 4;
     }
 
+    message OpAlterMeta {
+        required TabletMetaTypePB table_meta_type = 1;
```

Comment:
`table_meta_type` and `enable_persistent_index` are same right?

Reply:
This is to support modifications of more tablet meta type in the future. `tablet_meta_type` represents the type, and the other represents the specific value. Maybe it will be cleaner to change the name `TabletMetaTypePB` to `TabletMetaModifyType`.

Reply:
> This is to support modifications of more tablet meta type in the future. `tablet_meta_type` represents the type, and the other represents the specific value. Maybe it will be cleaner to change the name `TabletMetaTypePB` to `TabletMetaModifyTypePB`.

It seems a little wired. Maybe it is better that `TabletMetaTypePB` and its values are in a same message.

Reply:
> I think it should be the same. Even if type and value are in the same message, message `OpAlterMeta` still needs a type to indicate what type of alter it is.

Yeah. But I think they should be in the same message.